client/app/index.js

```diff
@@ -35,12 +35,9 @@ function loadModules() {
 }
 
 if (!global.Intl) {
-  require.ensure([], (require) => {
-    require('intl');
-    require('intl/locale-data/jsonp/en');
-    require('intl/locale-data/jsonp/zh');
-    loadModules();
-  }, 'intl');
-} else {
-  loadModules();
+  import(/* webpackChunkName: "intl" */ 'intl');
+  import(/* webpackChunkName: "intl" */ 'intl/locale-data/jsonp/en');
+  import(/* webpackChunkName: "intl" */ 'intl/locale-data/jsonp/zh');
```

Comment:
@kxmbrian Is the import here async ?  If it is then we will need to `load_modules` after import success.